sleap_chrome_remote_desktop/README.md

```diff
@@ -0,0 +1,154 @@
+# sleap-chrome-remote-desktop
+
+## Description
+This repo contains a DockerFile for a lightweight container (~9.75 GB) with the PyPI installation of SLEAP and all of its dependencies and a Chrome Remote Desktop server.
+
+The base image used is [eberrigan/sleap-cuda:latest](https://hub.docker.com/layers/eberrigan/sleap-cuda/latest/images/sha256-9cc93c86cc60d0f8e357bf58c2901d9b29a509c70ae16ed90ea56ac6d33418e7?context=repo).
+
+- The Dockerfile is located at `docker/Dockerfile`.
+- The repo has CI set up in `.github/workflows` for building and pushing the image when making changes.
+  - The workflow uses the linux/amd64 platform to build. 
+- `.devcontainer/devcontainer.json` is convenient for developing inside a container made with the DockerFile using Visual Studio Code.
+- Test data for training is located in `tests/data`.
+
+## Installation
+
+**Make sure to have Docker Daemon running first**
+
+
+You can pull the image if you don't have it built locally, or need to update the latest, with
+
+```bash
+docker pull eberrigan/sleap-chrome-remote-desktop:latest
+```
+
+## Usage
+
+Then, to run the image with GPUs interactively,
+
+``` bash
+docker run --gpus all -it eberrigan sleap-chrome-remote-desktop:latest
+```
+
+and test with
+
+```bash
+python3 -c "import sleap; sleap.versions()" && nvidia-smi
+```
+
+In general, use the syntax: 
+
+```bash
+docker run -v /path/on/host:/path/in/container [other options] image_name [command]
+```
+
+**Notes:**
+
+- `-it` ensures that you get an interactive terminal. The `i` stands for interactive, and `t` allocates a pseudo-TTY, which is what allows you to interact with the bash shell inside the container.
+- The `-v` or `--volume` option mounts the specified directory with the same level of access as the directory has on the host.
+- The `--rm` flag in a docker run command automatically removes the container when it stops. This is useful for running temporary or one-time containers without cluttering your Docker environment with stopped containers.
+
+To build the image locally, run
+
+```bash
+docker build -t sleap-chrome-remote-desktop .
+```
+
+Then, to run the image with gpus interactively
+
+```bash
+docker run --rm -it --gpus=all sleap-chrome-remote-desktop
+```
+
+# Connecting to the Chrome Remote Desktop
+
+This guide explains how to connect to the Chrome Remote Desktop server running in the container to access the graphical user interface (GUI).
+
+## Prerequisites
+
+1. **Google Account**: You need a Google account to use Chrome Remote Desktop.
+2. **Chrome Browser**: You need to have the Chrome browser installed on your local machine.
+3. **Docker Setup**: 
+   - Ensure Docker is installed on your local machine.
```

Comment:
Docker should be on the remote machine so that we pull the image and build the container on the remote machine and then connect to it from our local machine. We have just been testing this all on one machine but it should work with a remote machine != local machine